admin-dev/themes/new-theme/scss/components/layout/_header_toolbar.scss

```diff
@@ -4,7 +4,7 @@
   top: $size-header-height;
   left: $size-navbar-width;
   right: 0;
-  z-index: 501;
+  z-index: 1;
```

Comment:
Reduce z-index to 1. I think it was previously added because of: https://github.com/PrestaShop/PrestaShop/commit/fbc6444b22#diff-f9b1e580902f493368ce09e8a713d3a3